components/gitpod-protocol/src/protocol.ts

```diff
@@ -123,6 +123,11 @@ export namespace User {
         if (!!user.organizationId) {
             return false;
         }
+        // If a user has already been onboarded
+        // Also, used to rule out "admin-user"
+        if (!!user.additionalData?.profile?.onboardedTimestamp) {
```

Comment:
that's a weak signal, as it's free to change for everyone, no?

Reply:
I believe that is a totally fine approach.